data/hai/hai reveal 1 intro.txt

```diff
@@ -777,8 +777,8 @@ mission "Hai Leaks Response 4"
 			`	In an oddly human gesture that feels very deliberate, he places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
 				goto end
 			label "saved geocoris"
-			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy."`
-			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go, so if you have any business on-world here, please take your time."`
+			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy.`
+			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go though, so if you have any business on-world here, please take your time."`
```

Comment:
```suggestion
			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go, though, so if you have any business on-world here, please take your time."`
```

Even in speech, I literally (not ironically literally either, quite literally literally) cannot say this sentence without the pause. Probably because of the rhyme.